packages/app/src/cli/models/extensions/specifications/app_config_privacy_compliance_webhooks.test.ts

```diff
@@ -49,31 +110,91 @@ describe('privacy_compliance_webhooks', () => {
     })
   })
 
-  describe('reverseTransform', () => {
+  describe('reverseTransform with declarative_webhooks flag', () => {
     test('should return the reversed transformed object', () => {
       // Given
       const object = {
-        customers_redact_url: 'https://customer-deletion-url.dev',
-        customers_data_request_url: 'https://customer-data-request-url.dev',
-        shop_redact_url: 'https://shop-deletion-url.dev',
+        customers_redact_url: 'https://example.com/customer-deletion',
+        customers_data_request_url: 'https://example.com/customer-data-request',
+        shop_redact_url: 'https://example.com/shop-deletion',
       }
       const privacyComplianceSpec = spec
 
       // When
-      const result = privacyComplianceSpec.reverseTransform!(object)
+      const result = privacyComplianceSpec.reverseTransform!(object, {betas: [BetaFlag.DeclarativeWebhooks]})
 
       // Then
       expect(result).toMatchObject({
         webhooks: {
-          privacy_compliance: {
-            customer_deletion_url: 'https://customer-deletion-url.dev',
-            customer_data_request_url: 'https://customer-data-request-url.dev',
-            shop_deletion_url: 'https://shop-deletion-url.dev',
-          },
+          subscriptions: [
+            {
+              compliance_topics: ['customers/redact'],
+              uri: 'https://example.com/customer-deletion',
+            },
+            {
+              compliance_topics: ['customers/data_request'],
+              uri: 'https://example.com/customer-data-request',
+            },
+            {
+              compliance_topics: ['shop/redact'],
+              uri: 'https://example.com/shop-deletion',
```

Comment:
will the uri's here change to something like `uri: '/shop-deletion'` with [Alex's PR where relative paths are supported](https://github.com/Shopify/cli/pull/3402), or compliance webhooks should be unaffected here?